src/DicomWebClient/API/MimeMappings.cs

```diff
@@ -0,0 +1,91 @@
+/*
+ * Apache License, Version 2.0
+ * Copyright 2019-2020 NVIDIA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Dicom;
+using System.Collections.Generic;
+
+namespace Nvidia.Clara.Dicom.DicomWeb.Client.API
+{
+    public enum MimeType : short
+    {
+        Dicom = 0,
```

Comment:
you don't need to actually assign values to the members of an `enum`, that happens for you 😏

Reply:
Yes, I understand.  😁 Put the values because they are not continuous if you look down further but yes, I know they start at 0.